suites/luminous/ansible/tier-0.yaml

```diff
@@ -0,0 +1,64 @@
+#===============================================================================================
+#-------------------------------------
+#-----    Smoke Test suite     ------
+#-------------------------------------
+# Conf: conf/luminous/ansible/tier-0.yaml
+# Smoke test cases for
+#    - Install RHCS3 with lvm scenario
+#
+#===============================================================================================
+tests:
+   - test:
+      name: install ceph pre-requisites
+      config:
+         is_production: true
+      module: install_prereq.py
+      abort-on-fail: true
+
+   - test:
+      name: ceph ansible
+      polarion-id: CEPH-83571467
+      module: test_ansible.py
+      config:
+        is_mixed_lvm_configs: True
+        ansi_config:
+            ceph_origin: distro
+            ceph_stable_release: luminous
```

Comment:
This configuration has no impact due to the above line, would it make sense to remove it